src/GraphQl/State/Provider/ReadProvider.php

```diff
@@ -124,6 +120,9 @@ private function getNormalizedFilters(array $args): array
         $filters = $args;
 
         foreach ($filters as $name => $value) {
+            // Prevent numeric keys like `'1'`
```

Comment:
I wonder if `\is_string($name)` was made for preventing this known issue with array.

I don't have a report from PHPStan with the cast strategy